nordicsemi/__main__.py

```diff
@@ -1079,7 +1079,11 @@ def get_port_by_snr(snr):
               help='Flash connectivity firmware automatically. Default: disabled.',
               type=click.BOOL,
               is_flag=True)
-def ble(package, conn_ic_id, port, connect_delay, name, address, jlink_snr, flash_connectivity):
+@click.option('-mtu', '--att_mtu',
```

Comment:
We should standardize on using hyphens in option names instead of underscores. Click will automatically create the underlying variable with underscore anyway. I realize the options in the lines immediately above don't follow the naming convention, but it is the case most other places in the file.
```suggestion
@click.option('-mtu', '--att-mtu',
```